Kinvey/KinveyTests/NetworkStoreTests.swift

```diff
@@ -771,6 +771,48 @@ class NetworkStoreTests: StoreTestCase {
         }
     }
     
+    func testTranslateQueryCodable() {
+        let store = try! DataStore<PersonCodable>.collection(.sync)
```

Comment:
Why is a `(.sync)` store type in the `NetworkStoreTests` file?

Reply:
good call, let me move that do the `SyncStoreTests`